src/Enterspeed.Source.UmbracoCms/Data/Repositories/EnterspeedJobRepository.cs

```diff
@@ -105,6 +105,18 @@ public void Save(IList<EnterspeedJob> jobs)
             }
         }
 
+        public EnterspeedJob GetFailedJob(string entityId)
+        {
+            var schema = Database.FirstOrDefault<EnterspeedJobSchema>($"WHERE EntityId = {entityId} AND JobState = {EnterspeedJobState.Failed.GetHashCode()}");
```

Comment:
If possible I would not to have raw parameters inserted into a sql string, to avoid sql injection.

Other places in the repo we use linq expressions like this example from the GetFailedJobs method:

var failedJobs = Database.Query<EnterspeedJobSchema>()
                .Where(x => entityIds.Contains(x.EntityId) && x.JobState == EnterspeedJobState.Failed.GetHashCode())

Reply:
You are absolutely right. I dont know what exactly I was thinking here :D